recipes/diplib/build.sh

```diff
@@ -0,0 +1,16 @@
+set -euxo pipefail
+
+mkdir -p _build
+cd _build
+
+# Zlib is only a transitive dependency for libtiff; not needed
+cmake ${CMAKE_ARGS} ${SRC_DIR} \
+    -DPYTHON_EXECUTABLE="${PYTHON}" \
+    -DDIP_ENABLE_ZLIB=OFF \
+    -DDIP_ENABLE_FFTW=ON \
+    -DDIP_ENABLE_FREETYPE=ON \
+    -DDIP_BUILD_DIPVIEWER_JAVA=OFF \
+    -DDIP_BUILD_JAVAIO=OFF
+
+make -j${CPU_COUNT} install
+make pip_install
```

Comment:
Seems like the python module build is easily separated from the CXX shared libraries build and upstream advertises the CXX interface. Is there a reason that the python modules aren't a separate output (diplib-python)? (pydip is already claimed on PYPI by another package).

Reply:
Mostly laziness :D But seeing that https://github.com/conda-forge/staged-recipes/pull/19667 might need the same split, I think I'll try splitting it. Thanks for the push ;)

Reply:
Done!